network/transport/grpc/outbound_connector.go

```diff
@@ -150,7 +150,7 @@ func (c *outboundConnector) tryConnect() (*grpcLib.ClientConn, error) {
 	if c.tlsConfig != nil {
 		dialOptions = append(dialOptions, grpcLib.WithTransportCredentials(credentials.NewTLS(c.tlsConfig))) // TLS authentication
 	} else {
-		dialOptions = append(dialOptions, grpcLib.WithInsecure()) // No TLS, requires 'insecure' flag
+		dialOptions = append(dialOptions, grpcLib.WithTransportCredentials(insecure.NewCredentials())) // No TLS, requires 'insecure' flag
```

Comment:
What is the reason for this change? Deprecation?

Reply:
correct, this is the recommended replacement